hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockInfoStriped.java

```diff
@@ -244,6 +244,11 @@ final boolean hasNoStorage() {
     return true;
   }
 
+  @Override
+  boolean isProvided() {
+    return false;
```

Comment:
Add a comment saying why it's never provided.

Reply:
Added the javadoc.